### ISSUE TITLE
Tags with classes not have linebreaks

### ISSUE THREAD

**Issue by Maxpain177** [babel/babel-sublime#295]
If tag not have className, it's normally inserts linebreak after pressing enter key.
But, if tag have className, linebreak don't inserts.

Without className:
```jsx
<div>|</div>
// Pressing Enter
<div>
    |
</div>
```
With className:
```jsx
<div className="class">|</div>
// Pressing Enter
<div className="class">
|</div>
```

**Comment by borela** [comment]
Try FJSX15 https://github.com/borela/naomi it even supports stage-0 features.

**Comment by Thom1729** [comment]
This should be fixed in v10.0.0-beta.1. See #379 for more information on the new release. Please report any problems with v10 in a new issue.
